web/ui/react-app/src/pages/alerts/AlertContents.tsx

```diff
@@ -80,6 +71,20 @@ const AlertsContent: FC<AlertsProps> = ({ groups = [], statsCount }) => {
         >
           Show annotations
         </Checkbox>
+        {alertKinds.map((alertKind, i) => {
+          return (
+            <Checkbox
+              defaultChecked
+              id={`${alertKind}-toggler`}
+              wrapperStyles={{ margin: '0 0 0 15px' }}
+              onClick={toggle(alertKind)}
+            >
+              <Badge color={['success', 'warning', 'danger'][i]} className="text-capitalize">
```

Comment:
This seems sliiiightly unclean because it couples the assumption of a length of 3 to the `alertKinds` list, but only implicitly. How about a map from state to color and iterating over that instead?

Reply:
Yep, will refactor this with a map, thanks!